server/src/main/java/org/elasticsearch/search/aggregations/metrics/AvgAggregationBuilder.java

```diff
@@ -81,7 +81,8 @@ protected void innerWriteTo(StreamOutput out) {
 
     @Override
     protected AvgAggregatorFactory innerBuild(QueryShardContext queryShardContext, ValuesSourceConfig config,
-                                              AggregatorFactory parent, Builder subFactoriesBuilder) throws IOException {
+                                              AggregatorFactory parent,
+                                              AggregatorFactories.Builder subFactoriesBuilder) throws IOException {
```

Comment:
:heart:

Reply:
Glad you like it.  Intellij stuffed in the fully qualified package name there, and I did a manual pass yesterday to trim them down to just `AggregatorFactories.Builder`.